static/app/components/replays/actionCategory.tsx

```diff
@@ -3,68 +3,52 @@ import styled from '@emotion/styled';
 
 import Tooltip from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
-import {
-  BreadcrumbType,
-  BreadcrumbTypeHTTP,
-  BreadcrumbTypeNavigation,
-  RawCrumb,
-} from 'sentry/types/breadcrumbs';
-
-import {convertCrumbType} from '../events/interfaces/breadcrumbs/utils';
+import {BreadcrumbType, Crumb} from 'sentry/types/breadcrumbs';
 
 type Props = {
-  category: RawCrumb;
-  description?: BreadcrumbTypeNavigation | BreadcrumbTypeHTTP;
+  action: Crumb;
+};
+
+type actionCategoryInfo = {
+  description: string;
+  title: string;
 };
 
-function getActionCategoryDescription(crumb: RawCrumb): string {
+function getActionCategoryInfo(crumb: Crumb): actionCategoryInfo {
   switch (crumb.type) {
     case BreadcrumbType.USER:
     case BreadcrumbType.UI:
-      return t('UI Click');
-    case BreadcrumbType.HTTP:
-      return t('Fetch');
+      return {
+        title: t('UI Click'),
+        description: `${crumb.category}: ${crumb.message}`,
+      };
     case BreadcrumbType.NAVIGATION:
-      return t('Navigation');
+      return {
+        title: t('Navigation'),
+description: `${crumb.category}: ${crumb.data?.from ?? ''} => ${crumb.data?.to ?? ''}`
+          (crumb.data && crumb.data?.to) || ''
+        }`,
+      };
```

Comment:
looks like lint is complaining about this line, here's what it should say:

```suggestion
      return {
        title: t('Navigation'),
        description: `${crumb.category}: ${crumb.data?.from ?? ''} => ${
          crumb.data?.to ?? ''
        }`,
      };
```